src/backend/marsha/core/migrations/0018_auto_20200603_0620.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.0.6 on 2020-06-03 06:20
+
+from django.conf import settings
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+def default_video_resolutions(apps, schema_editor):
+    """
+    Insert in the new video_sizes field default sizes
+    """
+    Video = apps.get_model("core", "Video")
+    Video.objects.update(resolutions=settings.VIDEO_RESOLUTIONS)
```

Comment:
Maybe a precaution "just in case" :man_shrugging: 
```suggestion
    Video.objects.filter(resolutions__isnull=True).update(resolutions=settings.VIDEO_RESOLUTIONS)
```